src/workers/parser/pausePoints.js

```diff
@@ -50,6 +51,52 @@ function getStartLine(node) {
   return node.loc.start.line;
 }
 
+// Finds the first call item in a step expression so that we can step
+// to the beginning of the list and either step in or over. e.g. [], x(), { }
+function isFirstCall(node, parentNode, grandParentNode) {
+  let children = [];
+  if (t.isArrayExpression(parentNode)) {
+    children = parentNode.elements;
+  }
+
+  if (t.isObjectProperty(parentNode)) {
+    children = grandParentNode.properties.map(({ value }) => value);
+  }
+
+  if (t.isSequenceExpression(parentNode)) {
+    children = parentNode.expressions;
+  }
+
+  if (t.isCallExpression(parentNode)) {
```

Comment:
It's a little hard to grasp the _why_ for some of this logic. Would this need to consider `parentNode.callee` for instance?

Reply:
Yeah, i feel like each line could get it's own comment. Will look